README.md

```diff
@@ -218,8 +218,6 @@ Use the below CSS variables to override the defaults:
 --spm-backdrop-background;
 ```
 
-For th
```

Comment:
or should this one actually be completed to a full sentence?

Reply:
No, it's a typo 👍